relay-server/src/services/buffer/envelope_store/mod.rs

```diff
@@ -1 +1,35 @@
+use relay_base_schema::project::ProjectKey;
+
+use crate::Envelope;
+
 pub mod sqlite;
+
+/// Struct that represents two project keys for an envelope.
+#[derive(Debug, Clone, Copy, Eq, Hash, Ord, PartialOrd, PartialEq)]
+pub struct EnvelopeProjectKeys {
```

Comment:
Decided to delete `StackKey` and introduce a generic object that has knowledge of `own` and `sampling` key that we can reuse.

Reply:
Two nits:

1. Having a plural in the name makes it awkward to work with, can we call it `ProjectKeyPair` or similar?
2. This is now defined in `envelope_store`, but it's used all across the `buffer` module. Should we move it up (either put it in `buffer/mod.rs` or in a separate `buffer/utils.rs` or similar)?

Reply:
Yes, I agree with both points. I wasn't happy about the place.